packages/netlify-cms-ui-default/src/styles.js

```diff
@@ -133,6 +133,7 @@ const buttons = {
     border: 0;
     border-radius: ${lengths.borderRadius};
     cursor: pointer;
+    background-color: ${colorsRaw.blueLight};
```

Comment:
I'm not so sure it's good to have a background color on the button root -- it looks like it is supposed to be generic. Let's just add the color to `FileWidgetButton` directly, like you did for the remove button.